### DISCUSSION TITLE
Welcome to TET Discussions!

### DISCUSSION THREAD

**Discussion by johnkeel-thork** [johnkeel-thork/TET#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ndias666** [comment]
HAppy to help with the app as you guys see fit ;) I'm QA manager in a iOS app with millions of users.

**Reply by johnkeel-thork** [reply]
Thank you!
I just setup a small text about the App objectives:
https://github.com/johnkeel-thork/TET/discussions/11
